tests/test_bgp_fact.py

```diff
@@ -0,0 +1,18 @@
+def test_bgp_facts(ansible_adhoc):
+    hostname = 'vlab-01'
```

Comment:
>hostname [](start = 4, length = 8)

Originally playbook could run on any DUT, now it is hardcoded.

Reply:
need to add dynamic inventory to handle the hostname, basically we need to input as a testbed name, and then we need to derive the host, ptf, and neighbor devices.